src/app/containers/Video/index.jsx

```diff
@@ -29,11 +34,13 @@ const VideoContainer = ({ blocks }) => {
   }
 
   const metadata = videoMetadata(aresMediaBlock);
-
   const captionBlock = filterForBlockType(blocks, 'caption');
-
   const nestedModel = deepGet(['model', 'blocks', 0, 'model'], aresMediaBlock);
-  const kind = deepGet(['subType'], nestedModel);
+  const kind =
+    deepGet(['model', 'blocks', 0, 'model', 'format'], aresMediaBlock) ===
```

Comment:
You can do `deepGet(['format'], nestedModel) === ...` here. Not applying as suggestion as it'll cause eslint errors.